stable/metallb/templates/controller.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  namespace: metallb-system
+  name: controller
+  labels:
+    {{- if .Values.helmLabels }}
+    heritage: {{ .Release.Service | quote }}
+    release: {{ .Release.Name | quote }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    {{- end }}
+    app: {{ template "metallb.name" . }}
+    component: controller
+  annotations:
```

Comment:
Make annotations configurable. You don't need them if you use the Prometheus Operator.